rpc/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+mod blob;
+pub mod client;
+mod error;
+mod header;
+mod share;
+
+pub use error::{Error, Result};
+
+pub mod prelude {
+    pub use celestia_proto::share::p2p::shrex::nd::Proof as RawProof;
+    pub use celestia_types::nmt::Namespace;
+    pub use celestia_types::Blob;
```

Comment:
i'm not sure if i like re-exporting types from celestia-types here.

Reply:
removed